detectors/gradle/gradle.go

```diff
@@ -250,9 +245,9 @@ func detectIncludedProjects(projectRootEntry gradleProjectRootEntry) (*includedP
 			} else {
 				log.TWarnf("Unable to find build script for %s", include)
 			}
-
-			projects.includedProjects = subprojects
```

Comment:
This was a bug: the Gradle project's subprojects were set to the last included project.